packages/extension-playground/package.json

```diff
@@ -9,9 +9,10 @@
   "license": "MIT",
   "private": true,
   "scripts": {
+    "analyze": "export ANALYZE_MODE=static && yarn bundle",
     "bundle": "tsc && webpack --config webpack.prod.config.js",
-    "develop": "webpack-dev-server --hot --disable-host-check --port 8080 --config webpack.dev.config.js",
-    "watch": "yarn lerna exec --scope @looker/extension-playground --stream 'BABEL_ENV=build babel src --root-mode upward --out-dir lib/esm --source-maps --extensions .ts,.tsx --no-comments --watch'"
+    "develop": "webpack serve --hot --disable-host-check --port 8080 --https --config webpack.dev.config.js",
+    "watch": "yarn lerna exec --scope @looker/extension-api-explorer --stream 'BABEL_ENV=build babel src --root-mode upward --out-dir lib/esm --source-maps --extensions .ts,.tsx --no-comments --watch'"
```

Comment:
is this change intentional?

Reply:
Nope. Copy/pasta. Removing watch as not needed.